### PR TITLE
Ge2

### PR DESCRIPTION
**Pull request by EIRM-Odoo**
Pull request without merge problems

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (eacm-odoo) [inline .vscode/settings.json]: add this file to the .gitignore

File: ge02_team03/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    'depends' : ['motorcycle_registry'],
+    'data' : [
+        'views/ge02_team03_menuitems.xml',
+        'views/ge02_team03_views.xml',
+    ],
+    'demo' : [
+
+    ],
+    'application' : False,
+}
```
Comment (eacm-odoo) [inline ge02_team03/__manifest__.py]: newline

File: ge02_team03/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    'license' : 'OPL-1',
+    'author' : 'Kawiil-Motors',
+    'website' : 'https://levm-odoo-team3-kawiil-motors-ge2-9076373.dev.odoo.com/',
+    'category' : 'Kauil/GE02',
+    'depends' : ['motorcycle_registry'],
+    'data' : [
+        'views/ge02_team03_menuitems.xml',
+        'views/ge02_team03_views.xml',
+    ],
+    'demo' : [
```
Comment (eacm-odoo) [inline ge02_team03/__manifest__.py]: remove if not use

File: ge02_team03/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+    'name' : 'ge02_team03',
+    'summary' : """Module to visualize registered motorcycles on a map""",
+    'description' : """Module to visualize registered motorcycles on a map""",
+    'license' : 'OPL-1',
```
Comment (eacm-odoo) [inline ge02_team03/__manifest__.py:5]: and the version?

**Review by eacm-odoo — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (eacm-odoo) [inline .vscode/settings.json]: why is this file still in the PR?

File: ge01_team03/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (eacm-odoo) [inline ge01_team03/__init__.py]: why PR of module GE2 has module GE1?

**Review by eacm-odoo — CHANGES_REQUESTED** [review]

File: ge01_team03/__init__.py (hunk 5 above)
Comment (EIRM-Odoo) [reply ge01_team03/__init__.py]: Because GE1 was already on main when the branch was updated

File: ge02_team03/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    'author' : 'Kawiil-Motors',
+    'website' : 'https://levm-odoo-team3-kawiil-motors-ge2-9076373.dev.odoo.com/',
+    'version': "0.1",
```
Comment (eacm-odoo) [inline ge02_team03/__manifest__.py]: 'version': "1.0.0",

**Review by eacm-odoo — APPROVED** [review]

**Pull request merged**